filament/backend/src/vulkan/platform/VulkanPlatform.cpp

```diff
@@ -675,13 +675,13 @@ Driver* VulkanPlatform::createDriver(void* sharedContext,
 
     VulkanContext context;
 
-    ExtensionSet instExts;
-    // If using a shared context, we do not assume any extensions.
+    // This constains instance extensions that are required for the platform, which includes
+    // swapchain surface extensions.
+    auto const& swapchainExts = getSwapchainInstanceExtensions();
```

Comment:
I think the proper way to do this is actually modify the api so that you'd pass the extensions enabled from the shared context.

Reply:
Does this update make sense to you? Instead of passing the extension strings directly in the shared context, the booleans representing the supported extensions would be passed through the shared context. These booleans would be set during creation of the instance and device that are also passed in the shared context.